code/modules/projectiles/guns/projectile/automatic/triage.dm

```diff
@@ -18,6 +18,7 @@
 	price_tag = 900
 	penetration_multiplier = 1.2
 	init_recoil = SMG_RECOIL(0.6)
+	gun_parts = null //she's a delicate, complicated lil flower. Don't try to pull her apart.
```

Comment:
have it give scrap metal or something null is bad!

Reply:
guns with null parts will give a special message about being too complicated to disassemble